package-lock.json

```diff
@@ -4486,9 +4486,9 @@
       }
     },
     "eslint-utils": {
-      "version": "1.4.2",
-      "resolved": "https://registry.npmjs.org/eslint-utils/-/eslint-utils-1.4.2.tgz",
-      "integrity": "sha512-eAZS2sEUMlIeCjBeubdj45dmBHQwPHWyBcT1VSYB7o9x9WRRqKxyUoiXlRjyAwzN7YEzHJlYg0NmzDRWx6GP4Q==",
+      "version": "1.4.0",
```

Comment:
Why the downgrade? /cc @kevinlee12 @ankita240796 

@kevinlee12 I think this is the problem with #7451 -- it's going to get clobbered. @ankita240796 I think we need to fix this ASAP.

Reply:
I'm not sure why this happened @seanlip